src/Sit.scala

```diff
@@ -21,14 +23,12 @@ class Sit(private val projectPath: String,
   /**
    * persist the current Model.Commit list to the directory, ideally it should be located in ./.sit
    */
-  private def persisToDisk: Unit = {
-    // TODO
+  private def persisToDisk(head: Commit): Unit = {
+    // TODO haven't finished yet...
     val fw = new FileWriter(sitConfigPath, true)
-    val uuid: UUID = randomUUID()
-    val form = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
-    val now = form.format(Calendar.getInstance().getTime)
 
-    val commitInfo: String = s"[$now] ${uuid.toString}\n"
+    val commitInfo: String = s"commit ${head.id} \n\tMessage: ${head.commitMessage} \n\tTimestamp: ${head.timestamp} \n\tAdded: ${head.diff.added}\n\tDeleted: ${head.diff.deleted}\n"
+

```

Comment:
I don't know if I should use json to save the Model.Commit list into .sit. I also want to confirm that we want to overwrite the whole list every time we got a new commit or just append the newest Commit to the list in .sit...

Reply:
Using json looks good to me.
For .sit we basically have no choice but overwrite the whole file every time.
For demo purposes, I don't see an issue there.